lib/bolt/config.rb

```diff
@@ -101,6 +102,14 @@ class Config
       "level"  => "`warn` for console, `notice` for file"
     }.freeze
 
+    APPLY_SETTINGS = {
```

Comment:
Since `apply` is a new top-level field in the config file, it should be added as an option in `Bolt::Config::OPTIONS`.